parking_permits_app/constants.py

```diff
@@ -26,3 +26,4 @@ class VehicleType(Enum):
 
 
 LOW_EMISSION_DISCOUNT = 50
+SECONDARY_VEHICLE_PRICE_INCREASE = 150
```

Comment:
This should be 50 also put some comment that its a percent  or update the name to say its is percent and not just some random number